packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -315,14 +304,14 @@ module.exports = (log, db, config, customs, push, mailer, subhub, profile) => {
         }
 
         await subhub.reactivateSubscription(uid, subscriptionId);
-        await db.reactivateAccountSubscription(uid, subscriptionId);
+
+        await updateSubscriptionsFromSubhub({ db, subhub, profile, uid });
```

Comment:
Similar comment here - I might just log any errors from `updateSubscriptionsFromSubhub` but not report as failures. The subhub calls are the most important, as they connect back the the source-of-truth in stripe.

> updateSubscriptionsFromSubhub never runs

Yeah, I was thinking these are not the only spots where `updateSubscriptionsFromSubhub` should be called. Also considering dropping calls to it into user login and maybe when the subscriptions management app gets loaded on payments-server. Basically, looking for more interesting spots to have a chance to fix-up any potentially inconsistent data just before it might have caused a problem. (e.g. immediately post-login is when an RP might look for subscription capabilities. loading up the subs management page is when the user is looking for updated subs)